package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-laying-hens-web",
-  "version": "1.49.4",
+  "version": "1.49.6",
```

Comment:
increase the version by 1. it should be 1.49.5

Reply:
@yunusfirat I did that because you already have the 1.94.5 in your PR https://github.com/DEFRA/ffc-grants-laying-hens-web/pull/94

Reply:
Your reference should always be the main branch. if you want to merge this ticket your build number should be 1.49.5.

Reply:
Version should always be one above main, regardless of however many tickets are open. This is just in case we decide to merge a different PR first etc.

It makes it more irritating when there are PRs constantly being merged, and therefore needing to update the version constantly, but means we can keep track of things more easily :)

Reply:
@Hozan94 I have merged 2 of my PR. You should merge the latest to your branch with a new build number. Thanks.

Reply:
@zavies97 @yunusfirat sure no problem.